chart/values.yaml

```diff
@@ -116,6 +116,17 @@ sync:
         all: false
         labels: {}
 
+# Integrations holds config for vCluster integrations with other operators or tools
```

Comment:
```suggestion
# Integrations holds config for vCluster integrations with other operators or tools running on the host cluster
```
Not sure if this suggestion narrows down the use case of `integrations` too much, but I could see that users might be confused how this is different to `external` then.

Reply:
Makes sense, external is basically configurations for tooling outside of vCluster and integrations are part of vCluster itself